src/handlers/userHandlers.js

```diff
@@ -20,7 +21,7 @@ export const getUserHandler = (token, userId) => {
 };
 
 export const changePasswordHandler = (token, userId, newPassword) => {
-  if (!username || typeof username !== 'string') {
+  if (!userId || typeof userId !== 'string') {
```

Comment:
Could we conceivably check for the length of the userId, or is it variable?

Reply:
I don't know about that part of the code. I just changed it because master appeared to have a bug (username wasn't found).

Besides that I dont think it should be a problem, because the backend checks the token before the password is changed

Reply:
I believe this validation is enough on FE. Backend has a middleware that checks whether the `userId` is actually valid Mongoose ID